lib/libgssapi/gss_accept_sec_context.3

```diff
@@ -463,12 +463,12 @@ others, and derivative works that comment on or otherwise explain it
 or assist in its implementation may be prepared, copied, published
 and distributed, in whole or in part, without restriction of any
 kind, provided that the above copyright notice and this paragraph are
-included on all such copies and derivative works.  However, this
-document itself may not be modified in any way, such as by removing
-the copyright notice or references to the Internet Society or other
-Internet organizations, except as needed for the purpose of
-developing Internet standards in which case the procedures for
-copyrights defined in the Internet Standards process must be
+included on all such copies and derivative works.
+However, this document itself may not be modified in any way, such as
+by removing the copyright notice or references to the Internet
+Society or other Internet organizations, except as needed for the
```

Comment:
These are ridiculous places to wrap these lines because of grep. You're going to split isoc into two lines? Really?

This is why you can't write doc with scripts.

Reply:
I originally wrote that as
```
included on all such copies and derivative works.
However, this
document itself may not be modified in any way, such as
```

but I thought people would object to the short `However, this` line.  I'm quite happy to submit that version if it's preferred.  (That would make the script considerably simpler!)

Also, this is boilerplate license text; very unlikely to attract grep.  And the phrase "Internet Society" appears in both the previous and subsequent paragraphs.

Reply:
People's use cases for doc can be incredibly diverse. See the resolution of the `man after midnight` easter egg.

I would fold after `way,` and then start the next newline with isoc.

Commas are a great place to wrap. Wrapping has a surprising amount of considerations. Here's an excerpt from a recent discussion on https://reviews.freebsd.org/D47178

"Line folding is kinda a whole thing (no change suggested, just fyi):

First, roff asks nothing goes over 80 characters unless there's an emergency (see #4).

Second, in a standard bsd console, the bsd editor nvi uses 8 characters to the left of the screen, so you have 72 characters before wrapping occurs. So personally, I like to limit it there, because then it's a logical extension of self-hosting: our doc is written with our tooling in mind. Of course, people are welcome to use whatever they like, but this is BSD culturally conscious best practice.

Third, the mdoc style guide for authors at mandoc.bsd.lv teaches that when lines are wrapped, they're as wrapped in as close to the middle as possible, for many reasons.

Fourth, keep in mind not breaking up phrases, or breaking them up in a logical way for grep, since grep searches a line at a time.

So, afaik those are all the main considerations when choosing how to wrap lines."